### ISSUE TITLE
dev mode in symfony 4.1.8 broken

### ISSUE THREAD

**Issue by ehymel** [symfony/symfony#29356]
Composer update today to update from symfony 4.1.7 to 4.1.8 breaks my site in dev mode only. The site works as expected in prod mode. I'm on apache 2.4.34. Any page on the site brings up error "503 Service unavailable". Nothing in var/log/dev.log. Apache logs say "Failed to read FastCGI header".

**Comment by bretrzaun** [comment]
Same here!

**Comment by emnsen** [comment]
Temporary hotfix for dev env. update composer.json below and run `composer update`.
```js
"symfony/debug": "4.1.7",
"symfony/event-dispatcher": "4.1.7",
"symfony/http-kernel": "4.1.7",
"symfony/http-foundation": "4.1.7",
"symfony/routing": "4.1.7"
```
I think maybe it can help until it's solved.

**Comment by emnsen** [comment]
Fixed in #29349.

**Comment by jc-oclock** [comment]
> I think maybe it can help until it's solved.

It helps, thanks.

**Comment by Ahmed-Ayman** [comment]
@emnsen :heart: 
this fixed my issue.
I was getting `[ERROR] The process has been signaled with signal "11".` whenever I run the dev server.

**Comment by ehymel** [comment]
This is fixed with symfony 4.2.0. Thanks all.
